src/main/java/stsjorbsmod/cards/wanderer/materialcomponents/MaterialComponentsDeck.java

```diff
@@ -16,6 +16,8 @@
 	resets completely for next fight.
  */
 public class MaterialComponentsDeck {
+    public static int playedCount = 0;
```

Comment:
It would be nice if this name were a little less ambiguous about "since when" - maybe `playedThisCombatCount` or something?